sweepai/handlers/on_ticket.py

```diff
@@ -394,7 +394,11 @@ def get_comment_header(index, errored=False, pr_message=""):
     # Random variables to save in case of errors
     table = None  # Show plan so user can finetune prompt
 
-    def edit_sweep_comment(message: str, index: int, pr_message=""):
+    def edit_sweep_comment(message: str, index: int, pr_message="", is_latest=False):
```

Comment:
can you make sure all of the calls to this are correct? i think this is wrong because is_latest will never be true